doc/conf.py

```diff
@@ -52,7 +52,7 @@
 
 # General information about the project.
 project = 'FOSHttpCache'
-copyright = '2014, FriendsOfSymfony'
+copyright = u'2014–2016, FriendsOfSymfony'
```

Comment:
why the "u"? could we just remove the year as its not covered+updated by styleci?